libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/model/RoutePoints.kt

```diff
@@ -10,5 +10,4 @@ import com.mapbox.geojson.Point
  */
 internal data class RoutePoints constructor(
     val stepPoints: List<List<List<Point>>>,
-    val flatList: List<Point>,
```

Comment:
1) we can remove this from kdoc;
2) do we need the data class at all in this case?